### ISSUE TITLE
VoiceOver announces “comma” although punctuation is set to “None”

### ISSUE THREAD

**Issue by kalairubachandrakumaran-oss** [digitalfabrik/integreat-app#4077]
### Describe the Bug
VoiceOver announces the word “comma” between the text and the checkbox.

### Steps to Reproduce

1. Activate screen reader (VoiceOver)
2. Go to malte -> Malte Rostock -> LSBTIQ* 
3. Navigate to the informational text (e.g. “Es kann einige Werktage dauern…”)
4. Move focus to the checkbox (“Ich habe die Datenschutzerklärung…”)

### Expected Behavior
VoiceOver should not announce any punctuation when the setting is set to “None”.

### Environment

- Device/Browser: iphone 12 mini (real device)
- OS version: 26.3.1
- App version: 4.3

### Related Issues
#3959 ; #3911

### Additional Information

<img width="63" height="218" alt="Image" src="https://github.com/user-attachments/assets/c9e41f1c-0af3-4c5c-b757-989fa0149c48" /><img width="63" height="218" alt="Image" src="https://github.com/user-attachments/assets/81ed9715-e99a-43e8-9e75-0e023faaa592" />

https://github.com/user-attachments/assets/bd96097c-782c-436c-a226-08daecb45320
